frontend/src/components/History/index.jsx

```diff
@@ -298,12 +435,12 @@ class History extends React.Component {
                     onPageSizeChange={(pageSize) => localStorage.setItem("pageSize", pageSize)}
                     SubComponent={row => this.getSubComponent(row)}
                     filterable={true}
-                    defaultSorted={[
-                        {
-                            id: "timestamp",
-                            desc: true
-                        }
-                    ]}
+                    // defaultSorted={[
```

Comment:
So I was thinking that it could be confusing with the unlabeled data since they don't have timestamps and so were showing up random. Maybe we should sort by timestamp and then text?

Reply:
I think that would be a good solution, how do you feel about it?